database/migrations/2024_04_16_115559_create_verb_state_events_table.php

```diff
@@ -10,18 +10,18 @@
     public function up()
     {
         // If they've already migrated under the previous migration name, just skip
-        if (Schema::hasTable($this->tableName())) {
+        if (Schema::connection($this->connectionName())->hasTable($this->tableName())) {
             return;
         }
 
-        Schema::create($this->tableName(), function (Blueprint $table) {
+        Schema::connection($this->connectionName())->create($this->tableName(), function (Blueprint $table) {
             $table->snowflakeId();
 
             $table->snowflake('event_id')->index();
 
             // The 'state_id' column needs to be set up differently depending
             // on if you're using Snowflakes vs. ULIDs/etc.
-            Id::createColumnDefinition($table, 'state_id')->index();
+            Id::createColumnDefinition($table, 'state_id')->connection($this->connectionName())->index();
```

Comment:
Why does the column definition need a `->connection()` call? Under the hood, this is just a call to `$table->id()` or whatever…

```suggestion
            Id::createColumnDefinition($table, 'state_id')->index();
```